tl/features/create_pseudo_gt.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+from tl.exceptions import RequiredColumnMissingException
+
+
+def create_pseudo_gt(df: pd.DataFrame, column_thresholds: list,
```

Comment:
Annotation of a parameter should be more specific,

column_thresholds: List[tuple(str, float)] something like this.But since you'll move the logic to this function (from the cli/create-pseudo-gt.py, it is not relevant here). For future reference